.gitignore

```diff
@@ -11,4 +11,9 @@ ccs_act_patching*.html
 *.html
 !data
 neel-plotly
-.vscode/
\ No newline at end of file
+.vscode/
+
```

Comment:
Can you please add .ipynb_checkpoints here and unstage the checkpoints in this PR?